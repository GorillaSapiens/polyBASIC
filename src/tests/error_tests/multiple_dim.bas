DIM a(3)
DIM a(4)
