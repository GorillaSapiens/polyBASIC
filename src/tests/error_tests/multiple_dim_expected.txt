SOURCE 2:5, ARRAY ❮a❯ ALREADY DEFINED ON LINE 1
